compiler/src/iree/compiler/Dialect/LinalgExt/IR/LinalgExtOps.cpp

```diff
@@ -1220,6 +1220,34 @@ LogicalResult ArgCompareOp::reifyResultShapes(
       .reifyResultShapes(b, reifiedReturnShapes);
 }
 
+SmallVector<AffineMap>
+IREE::LinalgExt::ArgCompareOp::getIndexingMapsForOperands() {
+  Builder b(getContext());
+  const int64_t rank = getInputRank();
+  return SmallVector<AffineMap>{b.getMultiDimIdentityMap(rank)};
+}
+
+SmallVector<AffineMap>
+IREE::LinalgExt::ArgCompareOp::getIndexingMapsForResults() {
+  MLIRContext *ctx = getContext();
+  const int64_t rank = getInputRank();
+  const int64_t redDim = static_cast<int64_t>(getDimension());
+
+  SmallVector<AffineExpr> proj;
+  for (int64_t i = 0; i < rank; ++i) {
+    if (i == redDim)
+      continue;
+    proj.push_back(getAffineDimExpr(i, ctx));
+  }
+
+  AffineMap resultMap = AffineMap::get(rank, 0, proj, ctx);
+  return SmallVector<AffineMap>{resultMap, resultMap};
```

Comment:
nit: you can return an initializer list
```suggestion
  AffineMap resultMap = AffineMap::get(rank, 0, proj, ctx);
  return {resultMap, resultMap};
```